sound/soc/intel/common/soc-acpi-intel-nvl-match.c

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: GPL-2.0-only
```

Comment:
Commit message says headset codec support but the commit only adds mockup devices.